datanode/gateway/graphql/schema.graphql

```diff
@@ -812,6 +812,16 @@ type Query {
     pagination: Pagination
   ): ReferralSetConnection!
 
+  "Get referrer fee and discount stats"
+  referralFeeStats(
+    "Optional market ID to filter for. If omitted, asset ID must be provided."
+    marketID: ID
```

Comment:
Where not perfect everywhere on this, so consider it non blocking but `marketId` lowercase 'd' is more consistent with the rest of the schema

Reply:
I had to fix a number of merge conflicts so I have updated this along with the assetId which also suffered from the same capitalisation.